core/tests/test_utils.py

```diff
@@ -1758,12 +1758,15 @@ def swap(self, obj, attr, newvalue):
         the generator will immediately raise StopIteration, and contextlib will
         raise a RuntimeError.
         """
-        original = getattr(obj, attr)
+        original = getattr(obj, attr, 'NO SUCH ATTRIBUTE')
```

Comment:
Should we make this string a global constant?

Reply:
Updated